regression/goto-analyzer/show-callsites/test.desc

```diff
@@ -0,0 +1,18 @@
+CORE
+main.c
+--show-callsites -
+^EXIT=0$
+^SIGNAL=0$
+__BY_ADDRESS__,f2,f4,f1,f5,f0
```

Comment:
How about: `^__BY_ADDRESS__,(f[01245],){4}(f[01245])$`? Then do similarly for the below.